src/PureCloud.Client/Contracts/IStationsApi.cs

```diff
@@ -4,7 +4,7 @@ namespace PureCloud.Client.Contracts;
 
 public interface IStationsApi
 {
-    Task DeleteStationAssociatedUserAsync(string stationId, CancellationToken cancellationToken = default);
+    Task<bool> DeleteStationAssociatedUserAsync(string stationId, CancellationToken cancellationToken = default);
```

Comment:
Document all the methods along with their parameters in the interface. In the implementation all you need to add is the inheritdoc